openstax_api.gemspec

```diff
@@ -25,10 +25,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'doorkeeper'
   s.add_dependency 'exception_notification'
   s.add_dependency 'openstax_utilities'
-  s.add_dependency 'lev', '>= 11'
+  s.add_dependency 'lev', '>= 12.0.1'
```

Comment:
I think it's `12.1.0` but the `>=` works anyway.

Reply:
oops good catch!